### PR TITLE
sci-biology/abyss: ebuild tweaks

### PR DESCRIPTION
**Pull request by mmokrejs**

### CODE REVIEW

**Comment by mmokrejs** [comment]
@trofi Would you mind bringing these changes over to abyss-2.0.2-r2 and 2.0.3?

File: sci-biology/abyss/abyss-2.0.2-r1.ebuild [hunk 1]
```diff
@@ -47,9 +52,19 @@ src_prepare() {
...
 
 src_configure() {
+	local myconf=""
 	# disable building haskell tool Misc/samtobreak
 	# unless request by user: bug #534412
 	use misc-haskell || export ac_cv_prog_ac_ct_GHC=
 
-	econf $(use_enable openmp)
+	if ! use sqlite; then
+		myconf="${myconf} --without-sqlite"
```
Comment (trofi) [inline sci-biology/abyss/abyss-2.0.2-r1.ebuild]: Can you use $(use_with) instead of `if ... fi`?

Comment (mmokrejs) [reply sci-biology/abyss/abyss-2.0.2-r1.ebuild]: @trofi Done. The boost DEPEND line could maybe be "improved" like in dev-python/graph-tool-2.23.ebuild ? Well, I do not know what that really does.

        `dev-libs/boost:=[context,python,${PYTHON_USEDEP}]`

**Comment by mmokrejs** [comment]
`git` diff d0e9f3c845d762117cb0df9170ae86d56b1cf0bf.. > /tmp/abyss-2.0.1-r1.ebuild.patch


[abyss-2.0.1-r1.ebuild.patch.txt](https://github.com/gentoo/gentoo/files/1836901/abyss-2.0.1-r1.ebuild.patch.txt)

I am sorry but I am not going to retry this, I do not have time. Please rework the patch as you wish.

**Comment by SoapGentoo** [comment]
Sorry, and I don't have the time to help people who are not willing to learn.

**Pull request closed without merging**